src/Mailchimp/Synchronisation/MemberRequest/AbstractMemberRequestBuilder.php

```diff
@@ -8,6 +8,7 @@ abstract class AbstractMemberRequestBuilder
 {
     private $email;
     private $isSubscribeRequest = true;
+    private $interests;
```

Comment:
I need use them in `build` method https://github.com/EnMarche/en-marche.fr/pull/5686/files/93b6729ddc732ad04c233b6a7dc2101171209407#diff-1027513da4df19f553fd1c164f9ae2ae469cb5258921193b0fdc18345ece3d43R48
And I preferred to add them to the abstract class and not overwrite the `build` method, because `interests` is a basic of MC like `mergeFields`.